docs/blog/ibis-version-6.0.0-release.ipynb

```diff
@@ -0,0 +1,533 @@
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "t.group_by(\"species\").agg(ibis._.species.count())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To reproduce the old behavior, you can rename the column to `count` with:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "t.group_by(\"species\").agg(count=ibis._.species.count())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Backends\n",
+    "\n",
+    "### Oracle\n",
+    "\n",
+    "The Oracle backend was added! See the [Voltron Data blog](TODO) for more details."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ibis.connect(f\"oracle://user:password@host\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### DuckDB\n",
+    "\n",
+    "There were various DuckDB improvements, but one notable new feature is the ability to attach to a SQLite database through DuckDB. This allows you to run OLAP queries via DuckDB significantly faster on source data from SQLite.\n",
+    "\n",
+    "First we'll create a DuckDB connection and show it has no tables:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "duckdb_con = ibis.connect(\"duckdb://\")\n",
+    "duckdb_con.list_tables()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then create a SQLite database with a table:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sqlite_con = ibis.connect(\"sqlite://penguins.db\")\n",
+    "sqlite_con.create_table(\"penguins\", t.to_pyarrow(), overwrite=True);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And attach it:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "duckdb_con.attach_sqlite(\"./penguins.db\")\n",
+    "duckdb_con.list_tables()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### MotherDuck support!\n",
+    "\n",
+    "MotherdDuck launched recently and is now supported in Ibis!\n",
+    "\n",
+    "Simply connect with the DuckDB backend using `md:` or `motherduck:` as the database."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ibis.connect(\"duckdb://md:\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Polars\n",
+    "\n",
+    "The Polars backend received many improvements from the community, with plenty of operations now supported.\n",
```

Comment:
I think we should recognize `mesejo` and `alexander-beedie` here

Reply:
Added some shoutouts -- let me know if I should adjust. Should we ping them here to let them know?